src/nitypes/waveform/_numeric.py

```diff
@@ -506,14 +513,14 @@ def channel_name(self, value: str) -> None:
         self._extended_properties[CHANNEL_NAME] = value
 
     @property
-    def unit_description(self) -> str:
+    def units(self) -> str:
         """The unit of measurement, such as volts, of the waveform."""
         value = self._extended_properties.get(UNIT_DESCRIPTION, "")
         assert isinstance(value, str)
         return value
 
-    @unit_description.setter
-    def unit_description(self, value: str) -> None:
+    @units.setter
+    def units(self, value: str) -> None:
         if not isinstance(value, str):
```

Comment:
This unrelated to the PR title and will get left out of the release notes. Please split this into a separate PR.